docs/examples/zoom.ipynb

```diff
@@ -129,8 +129,8 @@
    "source": [
     "providers = sans.providers + sans.isis.providers\n",
     "providers = providers + (\n",
-    "    sans.isis.transmission_from_background_run,\n",
-    "    sans.isis.transmission_from_sample_run,\n",
+    "    sans.common.transmission_from_background_run,\n",
+    "    sans.common.transmission_from_sample_run,\n",
```

Comment:
Maybe importing this top-level in `__init__.py` would make sense? `common` is meaningless to a user, but `sans.transmission_from_sample_run` can be understood?